Misc Other/Freetext_Airspace Bases.txt

```diff
@@ -166,7 +166,7 @@ N052.20.04.580:W000.19.45.226:Airspace Bases:FL75C
 ;Daventry CTA 22 C FL145-FL195
 N052.36.35.581:W000.42.20.407:Airspace Bases:FL145C
 ;Daventry CTA 23 C FL105-FL145
-N052.09.27.586:W001.53.37.245:Airspace Bases:FL105C
+N052.09.27.586:W001.53.37.245:Airspace Bases - Conditional Daventry CTA:FL105*C
```

Comment:
Whilst this is PERM, we note as conditional as the base is determined by Birmingham CTA 10 below (which is conditional)